src/tests/Microsoft.PowerFx.Core.Tests/CheckResultTests.cs

```diff
@@ -181,8 +182,33 @@ public void BasicParseErrors2()
 
             // Still assign some types
             var node = ((BinaryOpNode)parse.Root).Left;
-            var type = check.GetNodeType(node);
+            var type = check.GetNodeType(node);
             Assert.Equal(FormulaType.Number, type);
+        }
+
+        [Fact]
+        public void BasicParseErrors2_Decimal()
+        {
```

Comment:
This is basically the same as above, could be the same test but using [Theory] and [InlineData]

Reply:
Done.